Swift 3.1 compile warning — 'unsafeBitCast' from 'UnsafeRawPointer' to 'UnsafePointer<Int8>'

In the latest Xcode 8.3 beta, I'm getting the following error on line:

https://github.com/raphaelmor/Polyline/blob/master/Polyline/Polyline.swift#L171

```
let byteArray = unsafeBitCast((data as NSData).bytes, to: UnsafePointer<Int8>.self)
``` 

```
Polyline.swift:160:21: error: 'unsafeBitCast' from 'UnsafeRawPointer' to 'UnsafePointer<Int8>' gives a type to a raw pointer and may lead to undefined behavior
    let byteArray = unsafeBitCast((data as NSData).bytes, to: UnsafePointer<Int8>.self)
                    ^
Polyline.swift:160:21: note: use the 'assumingMemoryBound' method if the pointer is known to point to an existing value or array of type 'Int8' in memory
    let byteArray = unsafeBitCast((data as NSData).bytes, to: UnsafePointer<Int8>.self)
                    ^~~~~~~~~~~~~~                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                        .assumingMemoryBound(to: Int8.self)
Polyline.swift:160:21: note: use the 'bindMemory' method to assign type 'Int8' to uninitialized raw memory
    let byteArray = unsafeBitCast((data as NSData).bytes, to: UnsafePointer<Int8>.self)
                    ^~~~~~~~~~~~~~                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                        .bindMemory(to: Int8.self, capacity: <#capacity#>)
```

The compiler is suggesting it should be rewritten as:

```
let byteArray = (data as NSData).bytes.assumingMemoryBound(to: Int8.self)
```

Does this seem reasonable? I can open a pull request with these changes if desired.

--- a/Polyline/Polyline.swift
+++ b/Polyline/Polyline.swift
@@ -168,7 +168,7 @@ public func decodePolyline(_ encodedPolyline: String, precision: Double = 1e5) -
     
     let data = encodedPolyline.data(using: String.Encoding.utf8)!
     
-    let byteArray = unsafeBitCast((data as NSData).bytes, to: UnsafePointer<Int8>.self)
+    let byteArray = (data as NSData).bytes.assumingMemoryBound(to: Int8.self)
     let length = Int(data.count)
     var position = Int(0)
     

